doc/conf.py

```diff
@@ -383,5 +387,21 @@ def html_page_context(app, pagename, templatename, context, doctree):
         context["theme_use_edit_page_button"] = False
 
 
+class PatchedPythonDomain(PythonDomain):
+    """
+    Workaround for `WARNING: more than one target found for 'any' cross-reference`
+
+    source: https://github.com/sphinx-doc/sphinx/issues/3866
+    """
+
+    def resolve_xref(self, env, fromdocname, builder, typ, target, node, contnode):
+        if "refspecific" in node:
+            del node["refspecific"]
+        return super().resolve_xref(
+            env, fromdocname, builder, typ, target, node, contnode
+        )
+
```

Comment:
remove?? i added this thinking it was going to help with the cross-reference issue 


```suggestion
```